lib/linalg/svd/StaticSVD.h

```diff
@@ -128,9 +125,9 @@ class StaticSVD : public SVD
      *         date.
      */
     bool
-    thisIntervalBasisCurrent()
+    isBasisCurrent()
```

Comment:
The comments for this function are not "current." It would be good to explain what "current" means.